packages/frontend/src/pages/Login/twitterCallback.tsx

```diff
@@ -0,0 +1,53 @@
+import { PATHS } from '@/constants/paths'
```

Comment:
Please follow file name convention of react component. Update to `TwitterCallback.tsx`